server/game/cards/08.6-SAT/BoisSacre.js

```diff
@@ -0,0 +1,31 @@
+// https://i.redditmedia.com/2ZjgGkqjmt3xilgW4EyicxqikQOoh4WP6J-mq5cE150.jpg?w=495&s=1f409ec2df0388fcf2e19e11f90e8561
+
+const DrawCard = require('../../drawcard.js');
+
+class BoisSacre extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                afterChallenge: ({challenge}) =>
```

Comment:
There's a bunch of these in the code, but we try not to deconstruct event objects anymore, so ```event``` and ```event.challenge.winner``` instead.